spec/check-engines_spec.js

```diff
@@ -177,4 +185,33 @@ describe('check-engines', function() {
       });
     });
   });
+
+  describe('invalid engines', function() {
+    var json = require('./fixtures/invalid-engines.json');
+    var badCommandMock;
+    beforeEach(function() {
+      badCommandMock = createMockChildProcess();
+      childProcess.spawn.withArgs(
+        'this-is-not-an-executable', ['-v']
+      ).returns(badCommandMock);
+      process.nextTick(function() {
+        badCommandMock.emit('error', new Error('unable to execute command'));
+        mockChildProcess.stdout.emit('data', '2.11.2\n');
+        mockChildProcess.emit('close');
+      });
+    });
+
+    it('handles invalid ranges and engines', function(done) {
+      checkEngines(json, function(error) {
+        expect(error).not.to.equal(null);
+        expect(error.message).to.contain(
+          'Unable to determine version for (this-is-not-an-executable)'
+        );
+        expect(error.message).to.contain(
+          'does not satisfy specified range (not a valid range)'
+        );
```

Comment:
For the above tests, I'd prefer to have a single assertion per test. Keeping one of the assertions for `error.message` should be sufficient here.